vector/src/main/res/values/theme_dark.xml

```diff
@@ -25,6 +25,9 @@
         <!-- waiting view background -->
         <item name="vctr_waiting_background_color">#55555555</item>
 
+        <!--icon tint color-->
+        <item name="vctr_general_icon_tint_color">@android:color/white</item>
```

Comment:
Can't it has the same color than the text? New vctr attribute should not be needed then. The legacy code is quite unclear, with several xml files for the same thing...

Reply:
textColorPrimary are selectors basically . And tint attribute and selector make crash in API 16 . 
If want to match text color, only option is to give secondary or tertiary text colors.
Also I thought it would make more sense to make different attribute for icon colors ( same attribute could be used for future icons).